tree/tree_test.go

```diff
@@ -413,7 +413,6 @@ Root
 }
 
 func TestRootStyle(t *testing.T) {
-	lipgloss.SetColorProfile(termenv.TrueColor)
```

Comment:
We do want this test to output truecolor output. Since v2 always outputs colors this makes sense, but we shouldn't strip the ansi from the output (part of why I'm using golden file so it retains the ansi sequence)